packages/cozy-client/src/models/timeseries.js

```diff
@@ -64,16 +65,17 @@ export const saveTimeSeries = async (client, timeseriesOption) => {
  * Helper to retrieve time series by their date interval and source.
  *
  * @param {object} client - The CozyClient instance
- * @param {{ startDate, endDate, dataType, source, limit }} The query params.
+ * @param {TimeSeries} timeSeries - The query params.
```

Comment:
not sure this is a TimeSeries object. cc @paultranvan we need you here ;)

Reply:
No, those are parameters to find a timeserie: https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.timeseries.md

Reply:
By applying @Crash-- suggested changes from https://github.com/cozy/cozy-client/pull/964#discussion_r651791541 then there is no need to name the object anymore